docs/source/methods/geometry.rst

```diff
@@ -892,6 +896,44 @@ Dxy + Eyz + Fxz + Gx + Hy + Jz + K = 0`. Thus, the gradient to the surface is
     \\ 2Cz + Ey + Fx + J \end{array} \right ).
 
 
+.. _white:
+
+------------------------------
+White Boundary Conditions
+------------------------------
+The white boundary condition is usually applied in deterministic code, where the particle
+will hit the surface and travel back with isotropic angular distribution. Essentially, the
+change of particle's direction will comply to the cosine distribution instead of uniform 
+distribution. 
+The probability distribution function (pdf)  for the reflected direction can be expressed 
+as follows, 
+.. math::
+    : label: white-reflection-pdf
+
```

Comment:
I modified the documentation. However, I got an error msg when I try to use "make latex" or "make html" in docs/. Since I need to have a look at the updated version after I modified.